### PR TITLE
Bugfix mouseout click order

### PR DESCRIPTION
**Pull request by a11yup-old**
@sofatutor/development-team 

This is a patch for the jquery-ui-touch-punch lib.

tl;dr: I just reversed the order of mouseout and click events triggered when an touchend event happens.

We need this so the exercises behave correctly on touch devices, because some of them - like FindPairs - rely on the fact that on desktops, when the user clicks an item that click event must logically get fired before any mouseout event for that item. FindPairs needs this specifically because the color of an Item can change during the mouseover and mouseout events but before the click can actually persist the color. 

The original jquery-ui-touch-punch lib does first fire the mouseout event and only then the click event. That means the color first gets reset, as if the user just hovered out of the item and then the click gets triggered. This leads to connected but wrong colored items.

### CODE REVIEW

**Comment by a11yup-old** [comment]
I reviewed this with @siemieniec.
